app/assets/templates/team.html

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <div class="row text-center">
+    <div class="col-sm-3">
+      <p><img src="assets/Gerry400.jpg" class="img-circle team"/></p>
+      <p class="lead "><strong>Gerry Pass</strong></p>
+      <p>Senior Developer</p>
+    </div>
+    <div class="col-sm-3">
+      <p><img src="assets/Amanda400.jpg" class="img-circle team"/></p>
+      <p class="lead"><strong>Amanda Raymond</strong></p>
+      <p>Junior Developer</p>
+    </div>
+    <div class="col-sm-3">
+      <p><img src="assets/Chris400.jpg" class="img-circle team"/></p>
+      <p class="lead"><strong>Chris Markel</strong></p>
+      <p>Junior Developer</p>
+    </div>
+    <div class="col-sm-3">
+      <p><img src="assets/Kevin400.jpg" class="img-circle team"/></p>
+      <p class="lead"><strong>Kevin Abdo</strong></p>
+      <p>Project Manager</p>
```

Comment:
KDawgs going for a Product Manager gig and it's probably more in line with what he's doing.